examples/E2EBert/news_classifier.py

```diff
@@ -57,22 +58,24 @@ def __getitem__(self, item):
         """
         review = str(self.reviews[item])
         target = self.targets[item]
-
-        encoding = self.tokenizer.encode_plus(
+        encoded = self.tokenizer.encode_plus(
             review,
             add_special_tokens=True,
-            max_length=self.max_length,
```

Comment:
@shrinath-suresh  No, Actually as you know, when we were not getting the right embedding, I was trying with encoder instead of encode_plus. I am already working on updating this var as it was before now.